packages/@aws-cdk/aws-lambda-event-sources/lib/stream.ts

```diff
@@ -141,7 +149,7 @@ export abstract class StreamEventSource implements lambda.IEventSource {
       onFailure: this.props.onFailure,
       tumblingWindow: this.props.tumblingWindow,
       enabled: this.props.enabled,
-      filters: this.props.filters,
```

Comment:
the old behavior of `filters` has to work here still. So we cannot simply delete `filters` here.

Reply:
I will bring back `filters` but most of the tests has to be changed to `testDeprecated` to avoid the DeprecationError since any test with `addEventSource` will use the deprecated parameter